src/sentry/receivers/onboarding.py

```diff
@@ -388,6 +373,37 @@ def record_user_context_received(project, event, **kwargs):
 
 
 event_processed.connect(record_user_context_received, weak=False)
+
+
+def record_event_with_first_minified_stack_trace_for_project(project, event, **kwargs):
+    # We only want to record events from projects created after 2022-12-09,
+    # otherwise amplitude would receive a large amount of data in a short period of time
+    if project.date_added < datetime.datetime(2022, 12, 9, tzinfo=pytz.utc):
+        return
+
+    if not has_event_minified_stack_trace(event):
+        return
+
+    try:
+        user: APIUser = Organization.objects.get(id=project.organization_id).get_default_owner()
+    except IndexError:
+        logging.getLogger("sentry").warning(
+            "Cannot record first event for organization (%s) due to missing owners",
+            project.organization_id,
+        )
+        return
+
+    analytics.record(
```

Comment:
@priscilawebdev this is going to generate an analytics event for every event that has a stack trace. We only want to do it for the first event for each project. This is where adding a bit to `flags` on the `Project` model comes into play. As it is, this is going to blow up our analytics with way too many events 😬